modules/ROOT/pages/tokenization-formats.adoc

```diff
@@ -59,7 +59,7 @@ Using the example values of `4` for first and last respectively, means that the
 A decimal digit will be mapped into the Latin alphabet character range.
 
 [NOTE]
-You are limited to two illegal characters because you really only need one to make the data invalid (such as adding a letter A to a social security number). Adding additional invalid characters results in reducing the number of digits that can be used to randomize the data that is being tokenized.
+You are limited to two illegal characters because you really need only one to make the data invalid (such as adding a letter A to a social security number). Adding additional invalid characters results in reducing the number of digits that can be used to randomize the data that is being tokenized.
```

Comment:
2 per what unit of measure? per field? or...?
Put A in tick marks?

I don't really understand what this note is trying to tell me.